### DISCUSSION TITLE
echo not requiring parenthesis

### DISCUSSION THREAD

**Discussion by benstigsen** [zuri-lang/zuri#184] [Polls]
Long time since I've been here, but I've been following the project closely.
One thing that I have not been a fan of, is the exception with `echo` not requiring braces.
It's inconsistent and I'm not quite sure I understand the purpose of it?

**Comment by mcfriend99** [comment]
`echo` is a keyword and not a function. It's synonymous to a macro for:

```
print(x)
print('\n')
```

since this is a very common use case.

In Blade, keywords do not have braces.

It will seemed really weird (at least to me) to maintain two functions for printing object to screen.

**Comment by benstigsen** [comment]
I'd much prefer `println()` and `print()`,
`echo` just feels out of place.

`println(x)` being the equivalent of doing; 
```
print(x)
print('\n')
``` 
But nicely wrapped in `println()` which looks more in place.

**Comment by mcfriend99** [comment]
I think we can have a middle ground here. We can enable `println` built-in function and gradually fade out `echo`. But we'll need more people to vote on this before we can start.

**Comment by benstigsen** [comment]
That is fair

**Comment by ibnz36** [comment]
I'm really late! I recently came back to coding a bit after a long time, and I've been very interested in this project again. I definitely think `println()` would fit way more. It's simply harder to adapt to `echo`. That being said, if more keywords with similar functionalities were added, maybe `echo` wouldn't be that bad of an option. Any thoughts 6 months after?
